### DISCUSSION TITLE
How do I get specific property from databucket that is behind objects and arrays?

### DISCUSSION THREAD

**Discussion by Zcehtro** [mockoon/mockoon#897] [Help] [answered]
I have this in a databucket:

```javascript
{
  list: [
    {{#repeat 2}}
    {
      "idMessage": {{@index}},
      "date": "{{date '2022-01-01' '2022-12-12' "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"}}",
      "account": "{{faker 'finance.account'}}",
      "subject": "{{faker 'lorem.words' num: number=3}}",
      "message": "{{faker 'lorem.sentences' 2}}",
      "state": {{faker 'datatype.number' min=2 max=5 }}
    }
    {{/repeat}}
  ]
}
```
I want to grab `idMessage` from the second iteration.

I'm trying with `{{data 'messages' 'list.1.idMessage'}}` but I'm not having any success.

What is the correct way to use the data helper?

**Comment by 255kb** [comment, marked as answer]
Everything is correct, you just forgot the double quotes around the `list` property.

**Comment by Zcehtro** [comment]
Unbelievable!! Thank you so much!

**Reply by 255kb** [reply]
I couldn't believe it myself. I spent 15min trying to find why. It's late here, but I was really curious 😄
